README.md

```diff
@@ -13,18 +13,38 @@ The simplest way to get your Counterparty node up and running is to use Docker C
 sudo apt install docker-compose
 ```
 
-Then, for `mainnet`, run:
+Then run node services in background with:
 
 ```bash
-docker-compose -f simplenode/compose.yml up
+git clone git@github.com:CounterpartyXCP/counterparty-core.git
+cd counterparty-core
+mkdir ~/counterparty-docker-data
+docker-compose up -d
 ```
 
-For `testnet`, modify the Docker Compose file in `simplenode/` and then run:
+**To run a node you must have at least 1.5TB free.** By default all data is stored in the `~/counterparty-docker-data` folder. You can modify this folder with the environment variable `$COUNTERPARTY_DOCKER_DATA`. For example:
+
+```bash
+COUNTERPARTY_DOCKER_DATA=/var/data docker-compose up -d
+```
+
+Use `docker-compose logs` to view output from services. For example:
+
 ```bash
-docker-compose -f simplenode/compose.yml -p simplenode-testnet up
+docker-compose logs --tail=10 -f bitcoind
+docker-compose logs --tail=10 -f addrindexrs
+docker-compose logs --tail=10 -f counterparty-core
+```
+
+You can use the environment variable `COUNTERPARTY_NETWORK` to run a `testnet` node:
+
+```
+COUNTERPARTY_NETWORK=test docker-compose up -d
 ```
 
-Then wait for your node to catch up with the network. Note: this process currently takes a long time, beause it does not make use of the `bootstrap` or `kickstart` functionality. (See below.)
+NOTES:
+- By default, this Docker Compose script makes use of the `bootstrap` functionality, because Docker makes it hard to use `kickstart`. (See below.)
+- When working with a low-memory system, you can tell `addrindexrs` to use JSON-RPC to communicate with `bitcoind` using the environment variable `ADDRINDEXRS_JSONRPC_IMPORT`: `ADDRINDEXRS_JSONRPC_IMPORT=true docker-compose up -d`
```

Comment:
I think we shouldn't give users the option of not using `JSONRPC_IMPORT` because it'll be incompatible with `kickstart`

Reply:
sorry I don't understand.. you want to force people to use ADDRINDEXRS_JSONRPC_IMPORT=true ? It's a lot slower and I don't understand the connection with `kickstart`.

Reply:
so I assumed that if addrindexrs is reading directly from the leveldb files (when `JSONRPC_IMPORT=false`) that'll block kickstart from working because only one process can access leveldb at a time...

Reply:
`addrindexrs` is not reading from the leveldb files but only from the blk files and there is no lock problem on these files. Additionally addrindexrs only reads blk files for the first catch up, then it uses RPC.. and before kcikstart can begin addrindexrs must be up to date and therefore no longer read blk files anyway..

Reply:
okay gotcha!